docker/app/Dockerfile

```diff
@@ -0,0 +1,42 @@
+FROM php:7.0-apache
```

Comment:
IMO this should be reverted: we are using nginx in preprod and will be in prod, the Dockerfile is too sensitive to be changed like this.